### ISSUE TITLE
Why the total_loss is grow up?

### ISSUE THREAD

**Issue by ziippy** [tensorflow/models#6787]
I try the "Training a pet detector" in object detection page.
(https://github.com/tensorflow/models/tree/master/research/object_detection)

selected pre-trained model: faster_rcnn_resnet101_coco (in the tensorflow model zoo)

after start training and check the tensorboard

[DetectionBoxes_Precision]
![image](https://user-images.githubusercontent.com/19854237/57787220-f0690380-776f-11e9-944d-6dd5b53eec9f.png)

[DetectionBoxes_Recall]
![image](https://user-images.githubusercontent.com/19854237/57787250-fced5c00-776f-11e9-971e-5ada0543322f.png)

[Loss]
![image](https://user-images.githubusercontent.com/19854237/57787271-037bd380-7770-11e9-9386-cd6347a0abbd.png)

So.. why the total_loss is grow up?
and why the total_loss value is so big?

what's the matter?
Is my mistake?

please, help me.

**Comment by danyfang** [comment]
The total_loss is the validation loss and here if it is growing up, it suggests that your model might be overfitting. See [this](https://stackoverflow.com/questions/55992559/training-loss-value-is-increasing-after-some-training-time-but-the-model-detect) on stackoverflow.

**Comment by tensorflowbutler** [comment]
Hi There,
 We are checking to see if you still need help on this, as this seems to be an old issue. Please update this issue with the latest information, code snippet to reproduce your issue and error you are seeing.
 If we don't hear from you in the next 7 days, this issue will be closed automatically. If you don't need help on this issue any more, please consider closing this.
